js/src/tools/sanitizer.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * --------------------------------------------------------------------------
+ * Bootstrap (v4.3.0): tools/sanitizer.js
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * --------------------------------------------------------------------------
+ */
+
+const uriAttrs = [
+  'background',
+  'cite',
+  'href',
+  'itemtype',
+  'longdesc',
+  'poster',
+  'src',
+  'xlink:href'
+]
+
+// The allowedAttribute method should check to see if the supplied value is
+// a regular expression, this would allow this to support any number of aria
+// attributes by default and give users the ability to specify their own regular
+// expressions should the need arise (e.g. /^data-my-app-[\w-]*$/).
+const ARIA_ATTRIBUTE_PATTERN = /^aria-[\w-]*$/i
```

Comment:
This comment was more for the review, it should probably just say:

`Supply a regular expression pattern to detect aria-* attributes.`

Although the documentation could also mention that regular expressions can be passed as values for more custom needs.